tools/extra/hssi/ethernet/hssistats.py

```diff
@@ -37,6 +37,8 @@
 import mmap
 from ethernet.hssicommon import *
 
+# Sleep 50 millisceonds afte clearing stats.
```

Comment:
milliseconds after

Also, you're dividing by 1 million.  Wouldn't that be microseconds?

Reply:
Fix typos:
```
milliseconds after
```

Reply:
fixed